package.json

```diff
@@ -3,12 +3,12 @@
   "private": true,
   "description": "Essex PowerBI visual base",
   "scripts": {
-    "clean": "del-cli \"./packages/*/lib\" \"!./packages/visual-styling/lib\"",
+    "clean": "lerna run clean --stream",
     "lint": "tslint -t verbose \"./packages/*/{src,spec}/**/*.ts\"",
-    "compile": "lerna run compile --stream",
+    "compile": "lerna run compile --concurrency=1 --stream",
     "docs": "lerna run docs --stream",
     "mocha": "mocha --require ts-node/register \"./packages/*/{src,spec,lib}/**/*.spec.ts\"",
-    "test": "lerna bootstrap && npm-run-all --sequential clean lint compile mocha docs",
+    "test": "npm-run-all bootstrap clean lint compile mocha docs",
```

Comment:
Wont this run into issues if `clean` ends up completing after `compile`?

Reply:
npm-run-all is sequential by default